apps/citizen_advocate/config/config.exs

```diff
@@ -22,6 +22,11 @@ config :logger, :console,
   format: "$time $metadata[$level] $message\n",
   metadata: [:request_id]
 
+# additional keys `api_key` and `domain` are in .secret.exs
```

Comment:
I did a quick scan and I don't think that Twilio has any email offerings so I'm fine going with this.  We should probably have a "secrets sharing" session and exchange our various API keys (Mailgun, Propublica, & Twilio)